tests/ci/setup.sh

```diff
@@ -61,8 +61,21 @@ if [[ "${ID}-${VERSION_ID}" == "centos-8" ]]; then
     cp -fv /etc/redhat-release "${TMPCI_DIR}"
     cp -fv tests/ci/files/rhel-8-4-0-os-release /etc/os-release
     cp -fv tests/ci/files/rhel-8-4-0-rh-release /etc/redhat-release
-    cp -fv /usr/share/osbuild-composer/repositories/centos-stream-8.json /etc/osbuild-composer/repositories/
-    ln -sfv /etc/osbuild-composer/repositories/centos-stream-8.json /etc/osbuild-composer/repositories/rhel-8.json
+    #cp -fv /usr/share/osbuild-composer/repositories/centos-stream-8.json /etc/osbuild-composer/repositories/
+    #ln -sfv /etc/osbuild-composer/repositories/centos-stream-8.json /etc/osbuild-composer/repositories/rhel-8.json
```

Comment:
I do believe this is still needed as a workaround ...

Reply:
We do. I think the point of creating the yum-repo was for adding it as a new sources (as we already do with the neptune copr), not to use it instead of the default repos.
Mostly, because we aren't going to mirror the whole repository, just some packages. I think.

Reply:
@juanje yeah, this has changed. We are trying to provide the whole repo ourselves from the manifest file. @lioramilbaum is working on generating that repo, and the code @guy9050 is adding here will replace the repos completely. Will see what obstacles we hit with that.

Reply:
so is there any workaround needed here?

Reply:
Sorry, I wrote a few comments, but I didn't know that they were in some kind of draft :thinking: :sweat_smile: 
Be aware that now we have a temporary fix for the errors with the mirrors:
https://github.com/osbuild/automotive-ci/blob/main/tests/ci/setup.sh#L66